lib/os/__init__.py

```diff
@@ -139,7 +139,7 @@ def close(self, _):
     result = _encode_wait_result(state.Sys())
     if not result:
       return None
-    return state.Pid(), result
+    return state.status 
```

Comment:
Hmm looking at this a bit more it seems like state.Sys() (which is the same thing as state.status) is encoded the way we want (exit status in high byte and signal in low byte) on OS X so we could get rid of the _encode_wait_result() call and just use state.Sys() (which is already a number) directly.

Most commands will not be consistent across platforms for testing purposes, but I think the result of `exit N` for whatever N should work on most reasonable platforms for testing purposes.